scripts/destroy_no_terraform.sh

```diff
@@ -17,12 +17,21 @@ set -o errexit
 set -o pipefail
 set -o nounset
 
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+# If suffix env var isn't set (i.e. by build agent), use bootstrap output
+SUFFIX=${SUFFIX:="$(cd "${SCRIPT_DIR}"/../bootstrap/local && terragrunt output naming_suffix | tr -d '"')"}
```

Comment:
and you say you don't like bash 😉

Reply:
And I stand by that - I wrote this through gritted teeth :D